platform/mobile/ABMobileView.js

```diff
@@ -0,0 +1,105 @@
+const ABMobileViewCore = require("../../core/mobile/ABMobileViewCore.js");
+
+module.exports = class ABMobileView extends ABMobileViewCore {
+   // constructor(attributes, application, parent) {
+   //    super(attributes, application, parent);
+   // }
+
+   /**
+
+
+   /*
+    * @method componentList
+    * return the list of components available on this view to display in the editor.
+    * @param {bool} isEdited  is this component currently in the Interface Editor
+    * @return {array} of ABView objects.
+    */
+   componentList(isEdited) {
```

Comment:
Should this method be moved to core to reduce duplication? On initial glance this looks the same as in `platform_web`